### ISSUE TITLE
The description is about of an IEEE 1905.1 implementation but not an EasyMesh implementation

### ISSUE THREAD

**Issue by phpaiva** [prplfoundation/prplMesh-old#5]
In the EasyMesh spec V1.0 we find that all control instructions are indicated via CMDU/TLV. I quote page 40 of the spec:  

_"Multi-AP control messages are carried using the 1905 CMDU format as defined in [2]. The 1905 CMDU header includes a
Message Type field identifying the type of the message carried in the CMDU. 1905 Message Type values from the
reserved space are used for Multi-AP control messages. Multi-AP specific TLVs are defined using the 1905 tlvType value
range. "_

But the description (which seems to be the copy and paste of the description of just the1905.1 IEEE implementation found [here](https://github.com/BroadbandForum/WT-382)), talks about ALMEs, TCP transport and things which are totally optional to use in EasyMesh, as well any propietary development for the communication, because that interface is not defined. And I should say that the way proposed in that text is no EasyMesh compliant, because EasyMesh aims to use just one HLE-AL interface (which resides in the "controller" role), and then the control messages are propagated via CMDU/TLV.

In other words, the controller should cover the AL role, and send control messages to the agents (which covers an AL role too) by using custom CMDU/TLVs as defined in EasyMesh v1.0. 

So my question is... this proyect just covers an IEEE 1905.1 implementation with non EasyMesh compliant extensions? If you say no, are you saying that this framework could give me as a developer libraries to receive, parse, configure and send EasyMesh compliant CMDU messages by my own "Centralized Smart Mesh controller" ?

Thanks in advance,  and don't doubt in correcting me if you believe that I'm wrong or misunderstanding.

Best regards,

Pablo.

**Comment by arnout** [comment]
> But the description (which seems to be the copy and paste of the description of just the1905.1 IEEE implementation found [here](https://github.com/BroadbandForum/WT-382)), talks about ALMEs, TCP transport and things which are totally optional to use in EasyMesh, as well any propietary development for the communication, because that interface is not defined. And I should say that the way proposed in that text is no EasyMesh compliant, because EasyMesh aims to use just one HLE-AL interface (which resides in the "controller" role), and then the control messages are propagated via CMDU/TLV.

The fourth paragraph of README.md says:

_Below is (mostly) the README.md from the original Broadband Forum repository. It has not been kept up to date with recent changes in the code._

In other words, the README.md text does not cover current reality nor the direction in which we want to go.

prplMesh is only concerned with the AL entity.

In the current state, a few of the Multi-AP (aka EasyMesh (tm)) CMDUs and TLVs have been implemented, but still many more are to be done.

However, we are currently investigating the possibility of basing prplMesh on a full implementation of the Multi-AP specification that has been developed by Technicolor.
